mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,13 +3,21 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
+		var TPL_FAILURE_MESSAGE = '<div class="alert alert-error">' +
+				'{warningMessage} ' +
```

Comment:
@mateomustapic I think @markocikos will insist that you standardize naming here. **TPL_FAILURE**, **alert-error** and **warningMessage** - I can't go deep into code to see usages BUT in real life this will confuse developer that works on same code as you. It is important for you and others to produce nice and super clean code. Don't worry it is hard, you will learn it :)